lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
So the reason this function threw errors is because we basically want the app to crash. That's it, in a nutshell. 

Generally, you call the build function in your app initialization, and if the initialization fails for any reason, we just simply crash. Fail-fast behavior. There's no real sense in propagating the error up in a promise, because there's arguably no real way of recovering from it -- your app is in a bad, probably-unrecoverable state.

(If the need ever arises where we DO want to allow initialization to fail, then maybe we can add an opt-in parameter or callback or something. But I don't anticipate any of our projects wanting that.)